frontend/i18n/en.json

```diff
@@ -366,7 +366,8 @@
       },
       "network": {
         "subtitle": "What network are your validators on?"
-      }
+      },
+      "coming_soon": "Coming soon"
```

Comment:
**Request:** There is no need for this, we already have https://github.com/gobitfly/beaconchain/blob/0cc4ea4e81955f4b725c33e0b3cef2d1cec3ad9b/frontend/i18n/en.json#L46
Please delete the new addition and make use of the one in `common`.